src/index.css

```
@@ -66,3 +41,9 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+@theme {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)